notebooks/ldb-datacleaning.ipynb

```diff
@@ -0,0 +1,1782 @@
+{
```

Comment:
<p>I think you could explain why this is in santd by and isnert a link of the issue!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>